### PR TITLE
Add menu to kitchen

### PR DESCRIPTION
**Pull request by stephensxu**

### CODE REVIEW

File: db/migrate/20140913041141_change_kitchen_description_to_allow_null.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class ChangeKitchenDescriptionToAllowNull < ActiveRecord::Migration
+  def up
```
Comment (jfarmer) [inline db/migrate/20140913041141_change_kitchen_description_to_allow_null.rb:2]: Good!

File: db/migrate/20140913044101_add_menu_to_kitchens.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddMenuToKitchens < ActiveRecord::Migration
+  def change
```
Comment (jfarmer) [inline db/migrate/20140913044101_add_menu_to_kitchens.rb:2]: Good!

File: spec/models/kitchen_spec.rb [hunk 3]
```diff
@@ -44,7 +45,6 @@
     it { should_not allow_value("@123").for(:name) }
     it { should_not allow_value("1234t").for(:name) }
 
-    it { should validate_presence_of(:description) }
```
Comment (jfarmer) [inline spec/models/kitchen_spec.rb:47]: But shouldn't it validate presence of `:menu`, now? :)

Comment (stephensxu) [inline spec/models/kitchen_spec.rb:47]: Yes, gonna write more tests!

**Pull request merged**